foam_tokenwrapper/tests/pitzDaily/0/R

```diff
@@ -0,0 +1,52 @@
+/*--------------------------------*- C++ -*----------------------------------*\
```

Comment:
I think we should have some common instructions where the data needed by the unittests and examples resides. There might be also data which is not found on public repositories.